docs/mcp/index.md

```diff
@@ -47,7 +47,7 @@ MCP Toolbox provides out-of-the-box toolsets for the following databases and dat
 
 #### Google Cloud
 
-* BigQuery (including tools for SQL execution, schema discovery, and ML forecasting)
+* BigQuery (including tools for SQL execution, schema discovery, and AI-powered time series forecasting with the `forecast` tool)
```

Comment:
remove "with the forecast tool" - this is too much detail for a high-level summary. Notice that none of the other capabilities mention a specific tool name.